core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -3603,7 +3544,6 @@ class ReplicaManagerTest {
 
     // verify that broker 1 did remove its metrics when no longer being the leader of partition 1
     verify(mockTopicStats1).removeOldLeaderMetrics(topic)
-    verify(mockTopicStats1).removeOldFollowerMetrics(topic)
```

Comment:
This removal is legal as the method is a no-op actually. I have opened https://issues.apache.org/jira/browse/KAFKA-19406 to clean up.